src/main/resources/db/migration/V20210302__alter_business-service.sql

```diff
@@ -0,0 +1,5 @@
+alter table if exists business_service
+    add constraint UK_btcp3rc4skxn0oyxbbnok68do unique (name),
+    add constraint FK64f0d7vxcp60lyjlttb7dqk3t
+    foreign key (owner_id)
+    references stakeholder;
```

Comment:
@mrizzi if we add **unique** constraints then we will have problems identifying which **names** can not be used anymore. E.g. I've just made this test in the UI:

- Create a Business service with the name "myBusinessService".
- Delete the previous business service.
- Create a new Business service with the same name as step 1 "myBusinessService", The UI will show an error.

The error itself is correct because the name "myBusinessService" is already in the DB however it was "soft deleted" and the user is not able to see which are the "soft deleted" names so they will try to use them anyways.

Maybe we should think again about the impact in the UI and also in the DB if we decide to continue with the approach of "soft deletion"... As far as I know there are no explicit requirements for having unique names in some entities but for sure it is just a matter of time until it will become a requirement (at least for the UI)

Reply:
good point and it seems like @PhilipCattanach was thinking we should have this requirement about uniqueness for BS's names.
But, tbh, it's also -strictly speaking- out of scope for this PR, so I'll remove it.